recipes/hpfrec/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "hpfrec" %}
+{% set version = "0.2.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hpfrec-{{ version }}.tar.gz
+  sha256: e384feffe24858874a8e279f6107422501316f94590db7ea19ca69404ffa9397
+
+build:
+  skip: true  # [py<310 or win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - python
+    - cython >=3.0.0
+    - setuptools
+    - wheel
+    - pip
+    - python
+    - numpy >=1.25
```

Always pin exactly in host 
```suggestion
    - numpy 1.25
```